srcpkgs/dash/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'dash'
 pkgname=dash
 version=0.5.12
-revision=1
+revision=2
```

Comment:
This doesn't need a revbump, since when the `libedit` option is disabled (the default), the build is the same as before.